File: JAVA/UberVans.java
class UberVans extends Car{
    Map<String,Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVans(String license, Account driver,Map<String,ArrayList<String,Integer>> typeCarAccepted,ArrayList<String> seatsMaterial){
        super(license,driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: Python/uberVans.py
from car import Car

class UberVans(Car):
    typeCarAccepted = []
    SeatsMaterial = []

    def __init__(self, license, driver,typeCarAccepted,SeatsMaterial) -> None:
        super.__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.SeatsMaterial = SeatsMaterial